src/Tensor.jl

```diff
@@ -40,6 +40,13 @@ Tensor(data::Number; meta...) = Tensor(fill(data); meta...)
 
 Base.copy(t::Tensor) = Tensor(parent(t), labels(t); deepcopy(t.meta)...)
 
+function Base.copy(t::Tensor{T,N,SubArray{T,N,Array{T,M},I,L}}) where {T, N, M, I, L}
```

Comment:
Simpler yet equivalent definition.
```suggestion
function Base.copy(t::Tensor{T,N,<:SubArray{T,N}}) where {T, N}
```